### DISCUSSION TITLE
HELP ME please

### DISCUSSION THREAD

**Discussion by zhennaifada** [nishad2m8/BTC-Ticker#5] [General]
I have been stuck in this interface, and it is dynamic and rotating. It has confused me for two days.
![20241107223854](https://github.com/user-attachments/assets/0742a7a4-65d0-4277-9a0a-5d3c2d27e53a)

**Comment by nishad2m8** [comment]
> I have been stuck in this interface, and it is dynamic and rotating

Can you share serial debug data ?

**Comment by six-ddc** [comment]
I'm having the same issue, did you manage to solve it?
